lib/rcrm/types/object_attribute/insert_object_attribute.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'object_attribute'
+
+module RCRM
+  class InsertObjectAttribute < ObjectAttribute
+    def initialize(input, fields: nil)
+      super fields: fields, arguments: Arguments.new.input(input)
+    end
+
+    def type_name
```

Comment:
shouldn't need this... It should automatically pull it from the class name

Also, if you need to override the "default" value `self.type_name = xxxx` on the class should be enough instead of overriding the method